template.yaml

```diff
@@ -599,3 +599,81 @@ Resources:
       DomainName: !Ref ApiDomain
       RestApiId: !Ref NvaCristinProxyApi
       Stage: !Ref NvaCristinProxyApi.Stage
+
+  NvaCristinIntermediateStorageBucket:
```

Comment:
Will the karate tests be allowed to delete the stack after the tests has run? S3 bucket as far as i know need to be empty to be deleted...

Reply:
good point, as this bucket is "not important" I will add DeletionPolicy: Delete so that its deleted even if there is files there